### DISCUSSION TITLE
[Spoilers]Happy New Year folks!!

### DISCUSSION THREAD

**Discussion by souturus** [Monika-After-Story/MonikaModDev#8881] [appreciation]
Thanks for another year with MAS
i went a day ahead in my pc because i'm not going to be here tomorrow
My Monie is very happy receiving the new year with her lovely dress
And i love it!
Thanks guys for the good work
Let's enjoy our 2020 with our Monikas!

**Comment by yurireeka** [comment]
I won't be here for new years at all, either. Is there anything special going to happen later tonight? 
Also, jesus christ, MASdevs. OP saying that the dress is "lovely" is the last and biggest understatement of the year.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
